Formula/xray.rb

```diff
@@ -41,6 +46,44 @@ def install
     resource("geosite").stage do
       pkgshare.install "dlc.dat" => "geosite.dat"
     end
+
+    resource("example_config").stage do
+      pkgetc.install "vless_tcp_xtls.json" => "example_config.json"
+    end
+  end
+
+  def caveats
+    <<~EOS
+      An example_config.json is installed in #{etc}/xray.
+      For more, visit https://github.com/XTLS/Xray-examples.
+      `brew services` requires a config file at #{etc}/xray/config.json
```

Comment:
That means the install won't work out-of-the-box. I'd rather have the install create a config.json by default if it needs any (and none is present)

Reply:
To use its functionality (network proxy), manual configuration is required. The example_config.json is for demo purpose (not necessary for installation) and will not work.

Reply:
That's not the point. Installing the example config as `pkgetc/”config.json”` is better than telling the user to, say, copy the example file by themselves.

Reply:
But if the example file is installed as config.json and the user modified it, an update would overwrite the modification? How should this be fixed?

Reply:
It won't overwrite the config file if it's modified.

Reply:
Okay I didn't know that. Would an update modify geoip? So homebrew can distinguish user-modified file versus an outdated file, right? I made the changes.